### PR TITLE
BedwarsGameChangedStatusEvent

### PR DESCRIPTION
**Pull request by notfoundname**
## Description
new BedwarsGameChangedStatusEvent calls when game changes its status

### Types of changes
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

-   [x] My code follows the code style of this project.
-   [ ] My change requires a change to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.

### CODE REVIEW

File: api/src/main/java/org/screamingsandals/bedwars/api/events/BedwarsGameChangedStatusEvent.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.screamingsandals.bedwars.api.game.Game;
+
+/**
+ * @author Bedwars Team
+ */
+public class BedwarsGameChangedStatusEvent extends Event {
+	
+	private static final HandlerList handlers = new HandlerList();
```
Comment (iamceph) [inline api/src/main/java/org/screamingsandals/bedwars/api/events/BedwarsGameChangedStatusEvent.java:12]: Code style :P

File: api/src/main/java/org/screamingsandals/bedwars/api/events/BedwarsGameChangedStatusEvent.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    
+    /**
+     * @param game
+     */
+    public BedwarsGameChangedStatusEvent (Game game) {
+    	this.game = game;
+    }
+    
+	@Override
+	public HandlerList getHandlers() {
```
Comment (iamceph) [inline api/src/main/java/org/screamingsandals/bedwars/api/events/BedwarsGameChangedStatusEvent.java:23]: Another bad formating

File: plugin/src/main/java/org/screamingsandals/bedwars/game/Game.java [hunk 3]
```diff
@@ -1544,6 +1544,7 @@ public void run() {
             if (tick.getNextStatus() == GameStatus.RUNNING) {
                 BedwarsGameStartEvent startE = new BedwarsGameStartEvent(this);
                 Main.getInstance().getServer().getPluginManager().callEvent(startE);
+				Main.getInstance().getServer().getPluginManager().callEvent(statusE);
```
Comment (iamceph) [inline plugin/src/main/java/org/screamingsandals/bedwars/game/Game.java:1547]: Why two times?

Comment (iamceph) [reply plugin/src/main/java/org/screamingsandals/bedwars/game/Game.java:1547]: You have it two times everywhere, why?

Comment (notfoundname) [reply plugin/src/main/java/org/screamingsandals/bedwars/game/Game.java:1547]: fuck

**Comment by notfoundname** [comment]
Probably fixed,please check

**Pull request merged**